src/Main.java

```diff
@@ -1,5 +1,103 @@
 public class Main {
     public static void main(String[] args) {
 
+        // Задача 1
+        double sum = 0;
+        int finalSum = 2_459_000;
+        int monthlyAddition = 15_000;
+        int months = 0;
+        double interest = 1.01;
+        while (sum < finalSum) {
+            sum = (sum + monthlyAddition) * interest;
+            months++;
+            System.out.printf("Месяц %d, сумма накоплений равна %.2f рублей", months, sum);
+            System.out.println();
```

Comment:
```suggestion
            System.out.printf("Месяц %d, сумма накоплений равна %.2f рублей%n", months, sum);
```
можно просто вот так сделать, чтобы не писать `System.out.println()` отдельно